CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ If you are looking for the Ledger Releasing and Versioning Process then you can
 
 ## 8.8
 
+- Add the constructor BabbageNonDisjointRefInputs to BabbageUtxoPredFailure	
```

Comment:
Same here wrong changelog.

This should go into `cardano-ledger-babbage` changelog

Reply:
done